### ISSUE TITLE
[Feature Request] Loot Tables

### ISSUE THREAD

**Issue by IcarussOne** [TheCodex6824/ThaumcraftFix#2]
While not exactly a fix, it would be great to change the hardcoded drops on all the mobs except pechs and cultists (which do use loot tables) and have them utilize loot tables instead. That way I no longer would need to input a loot table myself using mixins and can just utilize this mod instead when it's out and it would benefit other potential addons as well. :)

Crates, urns, treasure bags (?), porous stone (?) and rifts would also benefit from this.

**List of Mobs without tables:**
-Angry Zombie
-Crimson Praetor (unless it utilizes the cultist table)
-Eldritch Construct
-Eldritch Crab
-Eldritch Guardian
-Eldritch Warden
-Firebat
-Furious Zombie
-Giant Taintacle
-Giant Taint Seed
-Greater Crimson Portal
-Lesser Crimson Portal
-Mind Spider
-Shambling Husk
-Taintacle
-Taint Seed
-Taint Swarm
-Thaumic Slime
-Wisp

**Comment by TheCodex6824** [comment]
Good idea, I think this makes a lot of sense to add. I don't think this would be too hard to do either - it's been a while since I made loot tables but I think it should just need a simple single method to be added to every mob class.
